demo/src/app/components/datepicker/demos/basic/datepicker-basic.ts

```diff
@@ -0,0 +1,19 @@
+import {Component} from '@angular/core';
+
+const now = new Date();
+
+@Component({
+  selector: 'ngbd-datepicker-basic',
+  template: require('./datepicker-basic.html')
+})
+export class NgbdDatepickerBasic {
+
+  model;
+
+  minDate = {year: 2000, month: 0, date: 7};
```

Comment:
Yeah, you're right, will rename, it will be better. Got confused by the standard js `Date` where there is a `day` field but `getDate()` method that returns it and `getDay()` that returns the day of the week...